packages/browser/__tests__/login/oidc/redirectHandler/AuthCodeRedirectHandler.spec.ts

```diff
@@ -145,8 +145,8 @@ function mockClientRegistrar(client: IClient): IClientRegistrar {
   };
 }
 
-const mockFetchOnce = (response: Response): void => {
-  window.fetch = jest.fn().mockReturnValueOnce(
+const mockFetch = (response: Response): void => {
```

Comment:
Since the cookie isn't set when we hit the `/session` endpoint, but when we request one of the user's resources, we need to mock two requests instead of one.

Reply:
Ah got it, thanks.